controlpanel/oidc.py

```diff
@@ -16,7 +16,7 @@ def create_user(self, claims):
         return User.objects.create(
             pk=claims.get("sub"),
             username=claims.get(settings.OIDC_FIELD_USERNAME),
-            email=claims.get(settings.OIDC_FIELD_EMAIL),
+            email=claims.get(settings.OIDC_FIELD_EMAIL, ''),
```

Comment:
The email address should always be present, but it is not very important to us, so just silently handle a missing value.